plugins/color_panel/ios/color_panel.podspec

```diff
@@ -0,0 +1,21 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html
+#
+Pod::Spec.new do |s|
+  s.name             = 'color_panel'
+  s.version          = '0.0.1'
+  s.summary          = 'temp fake color_panel plugin'
```

Comment:
Please provide a clearer description (here and for `description`) so that someone looking at this podspec would understand what this is. Something like "No-op implementation of color_panel to avoid build issues on iOS"